js/keyboard/keyboard.js

```diff
@@ -0,0 +1,147 @@
+import {
+	LANGUAGE,
+} from '../constants.js';
+import buttons from './buttonData.js';
+import Button from './button.js';
+
+export default class Keyboard {
+	constructor(outputEl, language) {
+		this.language = language || LANGUAGE.EN;
+		this.outputEl = outputEl;
+		this.buttons = [];
+		this.isShifted = false;
+		this.el = document.createElement('div');
+	}
+
+	draw() {
```

Comment:
The same as method `draw()` in class `Button`. Try to use templates and contact with DOM as rarely as possible.
Actually this method contacts with DOM too many times (`createElement` and especially `append` in the loop)